zh/deploy-on-gcp-gke.md

```diff
@@ -36,98 +34,112 @@ gcloud config set compute/region <gcp-region>
 
 使用以上命令，设置好你的 GCP 项目和默认的区域。
 
-### 创建 GKE 集群
+## 创建 GKE 集群和节点池
 
-1. 创建 GKE 集群和一个默认节点池：
```

Comment:
The ordered list is necessary for this section.

It helps users quickly pick out **steps** from intro/ref, etc.

Reply:
Agree